docs/Producing.md

```diff
@@ -18,6 +18,9 @@ topic: json-events
 ```
 
 For actually producing a single record, click on the `Produce record` link above the `PRODUCER` line; for producing 10 records, click on `Produce record x 10`.

```

Comment:
To produce a single record, click on the `Produce record` link above the `PRODUCER` line; to produce 10 records, click on `Produce record x 10`.

Reply:
fixed